upgrade_test.py

```diff
@@ -221,7 +222,9 @@ def _upgrade_node(self, node, upgrade_sstables=True, new_scylla_repo=None, new_v
 
         if self.params.get("enable_tablets_on_upgrade"):
             scylla_yaml_updates.update({"enable_tablets": True})
-
+        version = ComparableScyllaVersion(self.scylla_version)
+        if self.is_enterprise and version >= "2025.1.0~dev" or not self.is_enterprise and version >= "6.3.0~dev":
+            scylla_yaml_updates.update({"experimental_features": ["views-with-tablets"]})
```

Comment:
Do we need something similar in `_rollback_node`, too?

Reply:
I don't know, someone need to cross check it in the code 

Anyhow I would recommend appending to the flags, now the code is override anything that any might have put in there before

Reply:
We restore original scylla.yaml file. See following:
https://github.com/scylladb/scylla-cluster-tests/blame/c22fba4d/upgrade_test.py#L434